src/main/java/ua/ithillel/lesson4/ArraySizeException.java

```diff
@@ -0,0 +1,6 @@
+package ua.ithillel.lesson4;
+
+public class ArraySizeException extends RuntimeException {
+    public ArraySizeException(String s) {
+    }
```

Comment:
```suggestion
    super(message);
    }
```
тоді в Stack Trace буде виводитися ще твоє повідомлення